src/host/ft_host/API_OutputTests.cpp

```diff
@@ -758,10 +758,10 @@ void OutputTests::ReadConsoleOutputWWithClipping()
     adjustedRegion.Bottom += bufferSize.Y / 2;
 
     auto expectedRegion = adjustedRegion;
-    expectedRegion.Left = max(0, adjustedRegion.Left);
-    expectedRegion.Top = max(0, adjustedRegion.Top);
-    expectedRegion.Right = min(bufferSize.X - 1, adjustedRegion.Right);
-    expectedRegion.Bottom = min(bufferSize.Y - 1, adjustedRegion.Bottom);
+    expectedRegion.Left = std::max((SHORT)0, adjustedRegion.Left);

```

Comment:
There is no `short` literal in C++. I had to add the casting to make STL happy. Is there a better way?

Reply:
I believe you can `std::max<short>(0, adjustedRegion.Left)`

Reply:
Yeah it is better now. Still I'd love a way to represent `short` literal. I guess I'll have to wait until next birthday.

Reply:
FYI: If you use MSVC you can write `0i16`. Similarly `0ui16`. It just works.™
(That language extension has been in MSVC for ages now, so I guess you can consider it quite a stable feature.)